server/dash/dashboards/nc_summary_comparison.py

```diff
@@ -739,7 +742,7 @@ def update_figure(nc_dropdown):
         y=["counts", "nc_avg"],
         color_discrete_sequence=DISCRETE_COLORS,
         labels=LABELS,
-        title="Number of " + nc_dropdown +
+        title="Number of " + selected_nc +
         " Requests compare with the average of all Neighborhood Councils requests"
```

Comment:
"Requests compared with the average Neighborhood Council" ?

Reply:
Thanks for the catch - resolved.